### PR TITLE
Sscs 10138 new hearing channel types

### PR DESCRIPTION
**Pull request by omer-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10138

### Change description ###
Update HearingChannel enums in line with new refdata values

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by nickwest1 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/HearingChannel.java [hunk 1]
```diff
@@ -7,10 +7,11 @@
 @RequiredArgsConstructor
 public enum HearingChannel {
 
-    TELEPHONE("telephone", "Telephone", "Ffôn"),
-    VIDEO("video", "Video", "Fideo"),
-    FACE_TO_FACE("faceToFace", "Face To Face", "Wyneb yn wyneb"),
-    NOT_ATTENDING("notAttending", "Not Attending", null);
+    TEL("telephone", "Telephone", "Ffôn"),
```
Comment (MohammedHMCTS) [inline src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/HearingChannel.java:10]: Is the key not supposed to be updated no the enum name?

**Review by MohammedHMCTS — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/HearingChannel.java (hunk 1 above)
Comment (wmarshid) [inline src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/HearingChannel.java:10]: Is there a reason to have the enums named like this? From our perspective it's our code so we should call them something meaningful, unless I'm mistaken (like if you're calling getName() on them for example).

**Review by wmarshid — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/HearingChannel.java (hunk 1 above)
Comment (omer-hmcts) [reply src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/HearingChannel.java]: Corrected

Comment (omer-hmcts) [reply src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/HearingChannel.java]: Corrected

**Review by johnoneill46 — APPROVED** [review]

**Review by MohammedHMCTS — APPROVED** [review]

**Pull request merged**